config/settings.py

```diff
@@ -156,23 +156,30 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-if env("RENDER"):
-    ALLOWED_HOSTS.append(env("RENDER_EXTERNAL_HOSTNAME"))
-    DJANGO_SUPERUSER_USERNAME = env("DJANGO_SUPERUSER_USERNAME")
+if env('RENDER'):
+    ALLOWED_HOSTS.append(env('RENDER_EXTERNAL_HOSTNAME'))
+    DJANGO_SUPERUSER_USERNAME = env('DJANGO_SUPERUSER_USERNAME')
     DJANGO_SUPERUSER_PASSWORD = env(
-        "DJANGO_SUPERUSER_PASSWORD")
-    DJANGO_SUPERUSER_EMAIL = env("DJANGO_SUPERUSER_EMAIL")
-    AWS_ACCESS_KEY_ID = env("AWS_ACCESS_KEY_ID")
-    AWS_SECRET_ACCESS_KEY = env("AWS_SECRET_ACCESS_KEY")
-    AWS_STORAGE_BUCKET_NAME = env("AWS_STORAGE_BUCKET_NAME")
+        'DJANGO_SUPERUSER_PASSWORD')
+    DJANGO_SUPERUSER_EMAIL = env('DJANGO_SUPERUSER_EMAIL')
+    AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID')
+    AWS_SECRET_ACCESS_KEY = env('AWS_SECRET_ACCESS_KEY')
+    AWS_STORAGE_BUCKET_NAME = env('AWS_STORAGE_BUCKET_NAME')
     AWS_S3_CUSTOM_DOMAIN = f'{AWS_STORAGE_BUCKET_NAME}.s3.amazonaws.com'
     AWS_S3_OBJECT_PARAMETERS = {
         'CacheControl': 'max-age=86400',
     }
+    AWS_DEFAULT_ACL = 'public-read'
+    AWS_QUERYSTRING_AUTH = False
+    AWS_S3_FILE_OVERWRITE = False
+    STORAGES = {
```

Comment:
The custom user model is setup to pull from django.core.files.storage import default_storage. With this new change I think we will need to update the import so it's pointing to storages now. 

To recreate the error you can run `python manage.py test`. This should fail any test looking for default storage.

Reply:
Ah nice! I will run the tests and check it out, and update the PR accordingly. Thanks for the heads up @GitLukeW!